src/docs/015_tasks/03_tasks.adoc

```diff
@@ -5,7 +5,7 @@ include::_config.adoc[]
 
 include::../_feedback.adoc[]
 
-== At a Glance
+=== At a Glance
```

Comment:
@rdmueller this is the last _At a Glance_ section remaining. Should this one also be removed? It is the only one containing a complete tasks diagram.

Reply:
why not? let's remove it!